back/internal/pb/gogen.go

```diff
@@ -5,5 +5,5 @@ package pb
 //go:generate -command pb-gg protoc -I../../../msgs/proto -I../../../msgs/proto/vendor-extra/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis --grpc-gateway_out=logtostderr=true:.
 
 // generate grpc, grpc-gateway
-//go:generate pb-go ../../../msgs/proto/auth.proto ../../../msgs/proto/user.proto
-//go:generate pb-gg ../../../msgs/proto/auth.proto ../../../msgs/proto/user.proto
+//go:generate pb-go ../../../msgs/proto/auth.proto ../../../msgs/proto/user.proto ../../../msgs/proto/posts.proto
+//go:generate pb-gg ../../../msgs/proto/auth.proto ../../../msgs/proto/user.proto ../../../msgs/proto/posts.proto
```

Comment:
Please use singular for posts.proto.

Reply:
Done. I'll push my changes up with the rest of the requested changes.

Reply:
Done